components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt

```diff
@@ -18,3 +18,15 @@ fun String.toNormalizedUrl(): String {
     }
     return uri.toString()
 }
+
+/**
+ * Checks if this string is a URL.
+ */
+fun String.isUrl(): Boolean {
+    val trimmedUrl = this.trim { it <= ' ' }
+    if (trimmedUrl.contains(" ")) {
+        return false
+    }
+
+    return trimmedUrl.contains(".") || trimmedUrl.contains(":")
+}
```

Comment:
Heh, is that my implementation from Focus? :) There's still my javadoc in the Focus code base:
> TODO: This is a super simple and probably stupid implementation.

😂

Reply:
hehe, it's actually the same in Fennec too!

Reply:
Oh, interesting. I do not remember looking at Fennec; but if we came up with the same code then it's probably not that bad. :)